transport_nonce_tcp_test.go

```diff
@@ -1,13 +1,6 @@
 package dbus
 
-import (
-	"bufio"
-	"io/ioutil"
-	"os"
-	"os/exec"
-	"testing"
-)
-
+/* This seems to be broken. Commenting out for now.
```

Comment:
Do not just comment out tests. This test passes locally on my machine and in CI; it of course depends on having D-Bus installed. If it doesn't pass for some reason on your machine, please open a separate issue.